From c38b43a2a98eaf2dd3401998e982716e98339907 Mon Sep 17 00:00:00 2001
From: Murat Sezgin <msezgin@codeaurora.org>
Date: Tue, 27 Jan 2015 11:10:26 -0800
Subject: [PATCH 224/281] ipv6: Export ndisc_send_ns() function for public use.

This API is needed to implement a connection
manager for hardware accelaration.

Change-Id: Idcf5fa1d3d3151fa2a5e0c48c6b6cdd10fc335bf
Signed-off-by: Murat Sezgin <msezgin@codeaurora.org>
---
 net/ipv6/ndisc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/ipv6/ndisc.c b/net/ipv6/ndisc.c
index 3a553494ff16..f72673df75b9 100644
--- a/net/ipv6/ndisc.c
+++ b/net/ipv6/ndisc.c
@@ -665,6 +665,7 @@ void ndisc_send_ns(struct net_device *dev, const struct in6_addr *solicit,
 	if (skb)
 		ndisc_send_skb(skb, daddr, saddr);
 }
+EXPORT_SYMBOL(ndisc_send_ns);
 
 void ndisc_send_rs(struct net_device *dev, const struct in6_addr *saddr,
 		   const struct in6_addr *daddr)
-- 
2.17.1

